src/lib/pages/workflows-with-new-search.svelte

```diff
@@ -272,22 +271,13 @@
     </button>
   </div>
 </header>
-<Pagination
-  items={$workflows}
-  let:visibleItems
-  aria-label="recent workflows"
-  pageSizeSelectLabel={translate('per-page')}
->
-  <svelte:fragment slot="action-top-left">
-    <LabsModeGuard>
-      <svelte:fragment slot="fallback">
-        <WorkflowAdvancedSearch />
-      </svelte:fragment>
-      <WorkflowFilterSearch />
-    </LabsModeGuard>
-  </svelte:fragment>
-  <svelte:fragment slot="action-top-center">
-    <WorkflowDateTimeFilter />
-  </svelte:fragment>
-  <WorkflowsSummaryConfigurableTable workflows={visibleItems} />
-</Pagination>
+<div class="flex flex-row gap-2">
+  <LabsModeGuard>
+    <svelte:fragment slot="fallback">
+      <WorkflowAdvancedSearch />
```

Comment:
Some wonky things happen with the status filter if the height of the filter is greater than the table
|||
|--|--|
|<img width="517" alt="Screenshot 2023-08-02 at 12 07 05 PM" src="https://github.com/temporalio/ui/assets/15069288/b0241851-88d5-43a9-a2be-93cc93809af7">|<img width="400" alt="Screenshot 2023-08-02 at 12 07 47 PM" src="https://github.com/temporalio/ui/assets/15069288/561b45aa-07c0-49d9-bc2d-677e9fa5527b">|

And the input filters can get cut off a bit

<img width="672" alt="Screenshot 2023-08-02 at 12 11 46 PM" src="https://github.com/temporalio/ui/assets/15069288/d5fdf335-9102-4557-94e3-2d3007bfe2a9">